images/installer/root/usr/local/bin/user_setup

```diff
@@ -6,9 +6,15 @@ mkdir -p ${HOME}
 chown ${USER_UID}:0 ${HOME}
 chmod ug+rwx ${HOME}
 
+chown ${USER_UID}:0 ${HOME}/.ssh/config
+
 # runtime user will need to be able to self-insert in /etc/passwd
 chmod g+rw /etc/passwd
 
+# make required config dirs writable
+chown ${USER_UID}:0 ${HOME}/.kube/config
```

Comment:
These lines ignore `admin_kubeconfig_path`, and I think it will only work if `admin_kubeconfig_path` has its default value?